gluster/cli/peer.py

```diff
@@ -47,6 +47,32 @@ def detach(host):
     cmd = ["detach", host]
     return peer_execute(cmd)
 
+def detach_all():
+    """
+    Removes All Hosts from Cluster
+
+    :returns: nothing
+     print: success and failure messages, raises
+    """
+    peers = parse_peer_status(peer_execute_xml(["status"]))
+    errors_list = []
+    if len(peers) > 0:
+        print "Found "+str(len(peers))+" peers"
```

Comment:
Print is not required in library. Assume this library is imported in other application and and shows good summary of the status. But these print statements may spoil the output format by printing something which is not needed by the library caller.